apps/arena/priv/config.json

```diff
@@ -1162,33 +1162,49 @@
     }
   ],
   "items": [
-    {
-      "name": "golden_clock",
-      "radius": 200.0,
-      "effects": [
-        "golden_clock_effect"
-      ]
-    },
     {
       "name": "magic_boots",
       "radius": 200.0,
       "effects": [
         "magic_boots_effect"
-      ]
+      ],
+      "mechanics": {}
     },
     {
       "name": "mirra_blessing",
       "radius": 200.0,
       "effects": [
         "mirra_blessing_effect"
-      ]
+      ],
+      "mechanics": {}
     },
     {
       "name": "giant",
       "radius": 200.0,
       "effects": [
         "giant_effect"
-      ]
+      ],
+      "mechanics": {}
+    },
+    {
+      "name": "fake_item",
```

Comment:
I think this name is misleading, can we talk about renaming this item to bomb or something like that?

Reply:
As we talked with @manucamejo the name is right, since in the front will be displayed as an item rather than bomb